### ISSUE TITLE
Server vs client LR

### ISSUE THREAD

**Issue by gokulanv** [CS-savvy/experiment_with_tf_federated#1]
https://github.com/CS-savvy/experiment_with_tf_federated/blob/master/train_tff.py#L122

In this line, shouldn't it be client LR as we run the eval model on central server?
Can submit a  PR, if so.

**Comment by CS-savvy** [comment]
It doesn't matter, cause i am not using eval model for training. I was getting error while transfering weights from tff to keras model with un-compiled one. I will check if learning rate is required there or not and update accordingly.
thanks for pointing out :) @gokulanv
